client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -259,28 +245,22 @@ export default React.createClass( {
 	},
 
 	renderCurrentActionDropdown() {
-		const { isWpCom } = this.props;
-		let options = [];
-		let actions = [];
+		const options = [];
```

Comment:
wow, I didn't know you can push new elements in an array declared as constant! nice to know!